single-line-diagram/single-line-diagram-core/src/main/resources/animations.css

```diff
@@ -0,0 +1,32 @@
+/* ----------------------------------------------------------------------- */
+/* File : animations.css ------------------------------------------------- */
+/* feeder info  */
+.sld-cell-direction-top .sld-arrow-in {
+  offset-rotate: 0deg;
+  offset-path: path('M 0,-10 0,10');
+  animation: move var(--sld-arrow-animation-duration, 0s) linear infinite;
+}
+.sld-cell-direction-top .sld-arrow-out {
+  offset-rotate: 0deg;
+  offset-path: path('M 0,10 0,-10');
+  animation: move var(--sld-arrow-animation-duration, 0s) linear infinite;
+}
+.sld-cell-direction-bottom .sld-arrow-in {
+  offset-rotate: 0deg;
+  offset-path: path('M 0,10 0,-10');
+  animation: move var(--sld-arrow-animation-duration, 0s) linear infinite;
+}
+.sld-cell-direction-bottom .sld-arrow-out {
+  offset-rotate: 0deg;
+  offset-path: path('M 0,-10 0,10');
+  animation: move var(--sld-arrow-animation-duration, 0s) linear infinite;
+}
+@keyframes move {
+  to {
+    offset-distance: 100%;
+  }
+}
+/* Set arrows animation speed (1: the slowest -> 3: the fastest) */
+.sld-arrow-animation-low-speed { --sld-arrow-animation-duration: 3s }
+.sld-arrow-animation-nominal-speed { --sld-arrow-animation-duration: 2s }
```

Comment:
If you change the name from "nominal" to "average", it will have an impact here too!